manifest.yaml

```diff
@@ -0,0 +1,36 @@
+applications:
+- name: apiserver
+  memory: 512M
+  instances: 1
+  host: apiserver
+  path: api/
+- name: servicebroker
+  memory: 512M
+  instances: 1
+  host: servicebroker
+  path: servicebroker
+- name: eventgenerator
```

Comment:
we can push all the code in src directory while only only install the one we need.